f2/src/components/ErrorSummary/ErrorSummary.stories.mdx

```diff
@@ -19,9 +19,18 @@ import { Trans } from '@lingui/react'
 
 The ErrorSummary component uses final-form validation to assign errors to fields. This can be set in a `validate` function that returns an error object. The object is a list of field names as keys, pointing to lingui ids as values. This function is usually defined before exporting the form component.
 
+When you can have multiple errors on a single field, you can also pass an array to the errors object.
+
```

Comment:
Why are we changing these components?

Reply:
To accept multiple errors on field sets. Especially the date field set, where we can have multiple distinct errors.